packages/esm-outpatient-app/src/add-patient-toqueue/add-patient-toqueue-dialog.component.tsx

```diff
@@ -86,6 +87,7 @@ const AddVisitToQueue: React.FC<AddVisitToQueueDialogProps> = ({ visitDetails, c
           });
           closeModal();
           mutate();
+          mutateQueueEntries();
```

Comment:
should we have one mutate function to update the queue entries.

Reply:
One mutate is for the list with queue entries and the other is for people who have active visits and are not in the queue